Hunted Mobile/Hunted Mobile/Hunted Mobile/Service/Map/MapViewService.cs

```diff
@@ -122,7 +125,13 @@ public class MapViewService {
         private void SetPlayerPin() {
             playerPin = new Pin(MapView) {
                 Label = Player.UserName,
-                Color = Player is Thief ? (Player is FakePolice ? Xamarin.Forms.Color.Green : thiefPinColor) : policePinColor,
+                Color = Player is Thief ? thiefPinColor : policePinColor,
+            };
+            playerRadius = new Circle() {
+                FillColor = new Color(255, 0, 0, 0.2),
+                Center = new MapsuiPosition(Player.Location.Latitude, Player.Location.Longitude),
+                Radius = new Distance(20),
```

Comment:
We gebruiker nu in `MapViewModel` een const waarde, zodat het overal hetzelfde blijft. Als we die nu aanpassen, dan zal deze niet mee veranderen. Daarom zou ik de `const LOOT_PICKUP_MAX_DISTANCE_IN_METERS` doorgeven aan de `constructor` van MapViewService